README.md

```diff
@@ -1,86 +1,95 @@
-AMD Radeon ProRender delegate for USD (hdRpr)
+AMD Radeon ProRender rendering plugin for USD Hydra 
```

Comment:
Technically this is a "render delegate".  Maybe that would be a good way instead of plugin.

Reply:
It's actually a delegate, but technically it's a plugin. So maybe let's name it as USD team [names](https://graphics.pixar.com/usd/docs/api/hd_embree_page_front.html) embree delegate? What do you think?
> HdEmbree : Embree-based hydra renderer plugin.